src/PetriEngine/options.cpp

```diff
@@ -114,6 +114,10 @@ void printHelp() {
         "\n"
         "Options:\n"
         "  -smc <number of runs><depth>         Simulate a number of runs, with depth bound\n"
+        "  -i, <int transtions>                 Interesting Transitions:\n"
+        "                                       - SMCIT_ALL    Int trans for all trans\n"
+        "                                       - SMCIT_AM     Int trans for given marking\n"
+        "                                       - SMCIT_A      Int trans for a given p/n\n"
```

Comment:
If this is some guidance stuff it should probably be "Interesting transitions" and not Int trans

Reply:
```suggestion
        "  -i, <Interesting transitions>                 Interesting transitions:\n"
        "                                       - SMCIT_ALL    Int trans for all trans\n"
        "                                       - SMCIT_AM     Int trans for given marking\n"
        "                                       - SMCIT_A      Int trans for a given p/n\n"
```